pyrateoptics/raytracer/helpers.py

```diff
@@ -103,16 +103,14 @@ def build_pilotbundle(surfobj, mat, (dx, dy), (phix, phiy), Elock=None,
     param random_xy: (bool) choose xy distribution randomly?
     
     """
-    
```

Comment:
These lines were presumably accidentially removed. Please reinsert them into the code. Thank you :-)

Reply:
(the TODOs in line 107 -- 109)